Documentation/subcommands/config.md

```diff
@@ -0,0 +1,130 @@
+# rkt config
+
+This command prints the rkt configuration per stage in JSON format.
+
+## Structure
+
+The general structure is a simple hierarchy consisting of the following top-level elements:
+
+```
+{
+	"stage0": [...],
+	"stage1": [...]
+}
+```
+
+The entries "stage0" and "stage1" refer to stage-specific configuration. The generated output are valid configuration entries as specified in the configuration documentation.
+
+The "stage0" entry contains entries of rktKind "auth", "dockerAuth", and "paths". Note that the `config` subcommand will output separate entries per "auth" domain and separate entries per "dockerAuth" registry. While it is possible to specify an array of strings in the input configuration rkt internally merges configuration state from different directories potentially creating multiple entries.
```

Comment:
I'll make my plans about configuration clear here.

`stage0` configuration is the configuration read by the `rkt` binary. So, `rkt` needs to know the credentials for fetching images (`auth` and `dockerAuth` kinds), needs to know where to store the fetched images (`paths` kind) and needs to know which stage1 image should be used by default and where to look for it (`stage1` kind).

`stage1` configuration will keep the configuration read by the binaries inside stage1 image (in our case `init`, `enter` and so on). So in this entry, we would put networking configuration and other stage1 specific settings.

So, in the current PR, stage1 entry should be empty.